docs/getting-started/devenv/README.md

```diff
@@ -21,10 +21,10 @@ curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.38.0/install.sh | bash
 ```
 
 Quit and restart terminal
-Install the long-term support (lts) version of node.
+Install node version (14.18.1).
```

Comment:
I think we can keep this paragraph as is. See https://nodejs.org/en/about/releases/. Node 14 is considered Maintenance LTS release.